tests/test_domain_models.py

```
@@ -0,0 +1,121 @@
+        coords = Coordinates(latitude=Decimal("0"), longitude=Decimal("0"))
+        location = Location(id="loc1", name="Test", coordinates=coords)
+        sensor = Sensor(
+            id="s1",
+            location=location,
+            parameter=ParameterType.PM25,
+            unit=MeasurementUnit.MICROGRAMS_PER_CUBIC_METER
+        )
+        
+        measurement = Measurement(
+            sensor=sensor,
+            timestamp=datetime.utcnow(),
```

Ruff (0.12.2)

`datetime.datetime.utcnow()` used

(DTZ003)